FESS-GUI/conversionlib.h

```diff
@@ -0,0 +1,12 @@
+#ifndef CONVERSIONLIB_H
```

Comment:
I merged conversionlib.h with conversions.h it will show up when I push my changes.